.github/ISSUE_TEMPLATE/feature_request.md

```diff
@@ -0,0 +1,26 @@
+---
+name: Feature request
+about: Suggest an idea to discuss
+title: "[Feature Request]"
+labels: feature-request
+assignees: ''
+
+---
+
+_A new feature is some behavior that is not existing already. If what you describe is a slight variation of something already existing please use a enhancement request._
```

Comment:
There's no more enhancement request. This needs to be updated.

Reply:
done